### ISSUE TITLE
Confusion in the use of poseNet in real time

### ISSUE THREAD

**Issue by AlmitraB** [moberweger/deep-prior#23]
Hi. I am trying to reproduce your model in Tensorflow and Keras, however, I am confused about the use of poseNet. In the first phase, you train poseNet model type = 0, in real-time you load the trained model, but you call it with type = 11, which has not been previously trained, and ask it to predict.  I do not understand why we try to predict with a model that has not been trained, nor what was the purpose of training the previous one. I think maybe it is a kind of transfer learning, but I am not sure.
Please help me to understand.

Looking forward to your answers

**Comment by moberweger** [comment]
Thanks for using DeepPrior! The type=0 model is trained to output the PCA
subspace, whereas the type=11 model has the PCA components added to the
network as an additional layer. Note: PCA projection can be implemented as
a Dense/FC layer.
Hope that helps.

**Comment by AlmitraB** [comment]
Hi, Thank you very much for your reply. In my eagerness to understand the code I have re-read the article completely to contrast it with your code, and more questions have arisen. This one specifically is related to the graphs that are drawn and saved after the training of the Pose Estimation network.
It strikes me that the X-axis of the cost graph has values from 0 to 9000, I don't understand what kind of values it corresponds to. Similarly, in the error graph, I would like to know what the X-axis refers.

**Comment by moberweger** [comment]
The cost graph that you are referring to, plots the training loss (cost)
over the batches. Thus, when training for 100 epochs, the network is
trained using ~90 batches each 128 samples, leading to 9000 steps on the
x-axis. Hope that helps you to understand the code!

**Comment by AlmitraB** [comment]
Thank you very much for your help, I get the cost graph. Could you please explain me the validation error graph ? I think in this case it is along the epochs, but the number of epochs is 100 and the horizontal axis goes up to 50. What exactly is being plotted here ?

**Comment by moberweger** [comment]
The validation error is the mean Euclidean error over all joints in [mm].
